invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Sections/SectionServices.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Button, Card, Container, Header, Icon } from 'semantic-ui-react';
+
+export default class SectionServices extends Component {
+  render() {
+    return (
+      <Container fluid className={'dot-background-container'}>
+        <Container fluid className="dot-background">
+          <Container className={'fs-landing-page-section'}>
+            <Header
+              as={'h1'}
+              className={'section-header highlight'}
+              textAlign={'center'}
+            >
+              Our services
+            </Header>
+            <Card.Group itemsPerRow={4} stackable>
+              <Card className={'advert-card'}>
```

Comment:
What about making each card a method.

Reply:
I personally prefer JSX than `renderX` methods, I find them less readable in the code. But let's see how the rest of the team feels about that

Reply:
I don't like the methods either, I would just go for dump functional components.